Automattic/wp-calypso
# Plugins: Update Browser Plugin card typography

#### Proposed Changes

Update the typography of the following card items:
* Title
* Author
* Description
* Pricing

| Before  | After |
| ------------- | ------------- |
|<img width="529" alt="CleanShot 2023-01-20 at 14 02 27@2x" src="https://user-images.githubusercontent.com/5039531/213773937-22a70215-cc07-4ead-b386-759832d9ee39.png">|![CleanShot 2023-01-20 at 14 01 29@2x](https://user-images.githubusercontent.com/5039531/213773945-3fa4225a-2f69-451d-b21e-0f712f5a2f68.png)|


#### Testing Instructions
* Go to `/plugins`
* Check if the cards presents the same typographies as above

#### Pre-merge Checklist






Fixes #70901
